.github/workflows/deploy-environment.yml

```diff
@@ -189,66 +192,30 @@ jobs:
           container-name: bops
           image: ${{ steps.ecr-image.outputs.uri }}
 
-      - name: Deploy ${{ matrix.service_type }}
-        uses: aws-actions/amazon-ecs-deploy-task-definition@v1
-        with:
-          task-definition: ${{ steps[format('task-def-{0}', matrix.service_type)].outputs.task-definition }}
-          service: bops-${{ matrix.service_type }}-${{ inputs.environment-name }}
-          cluster: bops-${{ inputs.environment-name }}
-          wait-for-service-stability: true
-
-  deploy-web-service:
-    name: Deploy web service to ${{ inputs.environment-name }}
-    runs-on: ubuntu-20.04
-    needs: [build-image, deploy-db-migrate-service, deploy-services]
-
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
-
-      - name: Get github commit sha
-        id: github
-        run: |
-          echo "sha=$(echo ${GITHUB_SHA::7})" >>$GITHUB_OUTPUT
-
-      - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v2
-        with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: eu-west-2
-
-      - name: Login to Amazon ECR
-        id: login-ecr
-        uses: aws-actions/amazon-ecr-login@v1
-
-      - name: Get image URI
-        id: ecr-image
-        env:
-          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
-          ECR_REPOSITORY: bops/${{ inputs.environment-name }}
-          IMAGE_TAG: ${{ steps.github.outputs.sha }}
-        run: |
-          echo "uri=$ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG" >>$GITHUB_OUTPUT
-
-      - name: Download task definition for web and strip unused properties
-        run: |
-          aws ecs describe-task-definition --task-definition bops-web-${{ inputs.environment-name }} --query taskDefinition | \
-          jq -r 'del(.compatibilities, .taskDefinitionArn, .requiresAttributes, .revision, .status, .registeredAt, .registeredBy)' > web.json
-
       - name: Generate task definition for web
```

Comment:
I think we can probably also run this only once and use the `matrix.service-type` variable if we can figure out the right syntax for referring to the output. I'll experiment.